src/tests/GfxBatchHbaoTest.cpp

```diff
@@ -39,70 +39,130 @@ namespace Cr = Corrade;
 namespace Mn = Magnum;
 using namespace Mn::Math::Literals;
 
+// Change this if we want to test on a different core mesh. Requires
+// regenerating ground truth images.
 constexpr const char* baseTestFilename = "van-gogh-room";
 
-const std::string SourceTestPlyDir =
+// Location of test scene mesh from habitat_test_secenes repository.
+const Cr::Containers::String testPlySourceDir =
     Cr::Utility::Path::join(SCENE_DATASETS, "habitat-test-scenes");
 
-const std::string TestHBAOImageDir =
+// Validation images location (held in repository
+// TODO : consider moving these images to habitat_test_secenes repository.
+const Cr::Containers::String testHBAOImageDir =
     Cr::Utility::Path::join(TEST_ASSETS, "hbao_tests");
 
+// Size of images to test for validation
 constexpr Mn::Vector2i Size{320, 240};
 
-constexpr const float aspectRatio = float(Size.x()) / Size.y();
+// Aspect ratio of given size
+constexpr const float AspectRatio = float(Size.x()) / Size.y();
```

Comment:
Floating-point `Vector2` has [aspectRatio()](https://doc.magnum.graphics/magnum/classMagnum_1_1Math_1_1Vector2.html#a4f097e00b03c17ffe212da2c8371d614). Though it isn't `constexpr` for some reason, but that doesn't really matter in a test.

```suggestion
const float AspectRatio = Mn::Vector2{Size}.aspectRatio();
```